cloud/aws/services/ec2/errors.go

```diff
@@ -0,0 +1,8 @@
+package ec2
+
+import "errors"
+
+var (
+	// TODO(vincepri): gather more information like vpc arn.
+	ErrVPCNotFound = errors.New("vpc not found")
```

Comment:
I would prefer errors to be defined more like how they are described here so we can use functions to check if it's a type of error we expect instead of relying on string matching. https://blog.golang.org/error-handling-and-go

Reply:
Yep, agree. Just wanted to get something up quickly to share the initial shape and get some early comments.